src/Shared/Http2/CopyToCoreFx.cmd

```diff
@@ -0,0 +1,12 @@
+@ECHO OFF
+SETLOCAL
+
+IF "%COREFX_REPO%" == "" (
```

Comment:
These scripts should be able to take an absolute path to the place to copy to, not just an environment variable.

Reply:
What's the usage scenario for an absolute path?

The env variable is for inner loop development.

Reply:
I have multiple dotnet/aspnetcore repos I copy to and from. I have at least two clones of aspnetcore I work on regularly. Also it's more analogous to cp commands in unix.